src/app/shared/components/header/header.component.html

```diff
@@ -1 +1,15 @@
-<span>header works!</span>
+<mat-toolbar color="primary" class="header_toolbar">
+	<div class="header_toolbar__logo">
+		<img src="assets/images/logo.png" alt="Logo" class="logo" />
+		<span class="header_title">Meta Collectioner</span>
+	</div>
+
+	<div class="header_toolbar__actions">
+		<button mat-button (click)="goToHome()">Home</button>
```

Comment:
routerLink instead of binded methods are simpler
e.g. https://github.com/discrimine/meta-collectioner/blob/master/src/app/profile/profile.component.html#L9